app/content/deployment-options/custom-deployment/operations.md

```diff
@@ -1,5 +1,49 @@
 ## Operations
 
+### Configuring proxy
+
+When the instance is going to be installed behind a `proxy` it is necessary to:
+
+* Configure the `noProxy` policy for Docker:
+	```bash
+	# /root/.docker/config.json
+	{
+		"auths": {},
+		"credHelpers":
+		{
+      			"asia.gcr.io": "gcr",
+       			"eu.gcr.io": "gcr",
+       			"gcr.io": "gcr",
+       			"marketplace.gcr.io": "gcr",
+       			"us.gcr.io": "gcr"
+		},
+		"proxies": {
+      			"default":
+       			{
+       				"httpProxy": "<PROXY_ADDRESS>",
+       				"httpsProxy": "<PROXY_ADDRESS>",
+       				"noProxy": "127.0.0.1,localhost,<CARTO_DOMAIN>"
```

Comment:
are `<CARTO_DOMAIN>` and `<PROXY_ADDRESS>`  a placeholder that the user will need to replace with their actual values? If so, I would use an example that looks real and add a clarification note that makes it clear that they need their actual values.

Reply:
New description on the last commit (not completely sure on how to make changes according to comments 😂 )